dev/com.ibm.ws.messaging.open_jms20security_fat/test-applications/mdbapp/src/mdb/EJBAnnotatedMessageDrivenBean.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * IBM Confidential
```

Comment:
Please remove this ASAP!

Reply:
Please pay extra attention when porting code from WebSphere Liberty.

Reply:
Sure, thanks!

Reply:
The change is in.